### PR TITLE
Preview theme workflow fix

### PR DESCRIPTION
**Pull request by Zo-Bro-23**
Preview theme workflow fails. Hopefully fixes it.

### CODE REVIEW

**Comment by Zo-Bro-23** [comment]
@rickstaa again 😞

File: scripts/preview-theme.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ const REQUIRED_COLOR_PROPS = ACCEPTED_COLOR_PROPS.slice(0, 4);
 const INVALID_REVIEW_COMMENT = (commentUrl) =>
   `Some themes are invalid. See the [Automated Theme Preview](${commentUrl}) comment above for more information.`;
 
+// Retrieve octokit instance.
+const OCTOKIT = github.getOctokit(getGithubToken());
+const pullRequestId = prNumber ? prNumber : getPrNumber();
```
Comment (rickstaa) [inline scripts/preview-theme.js:49]: Maybe we should here also use the CONST convention?

```suggestion
const PULL_REQUEST_ID = prNumber ? prNumber : getPrNumber();
```

File: scripts/preview-theme.js [hunk 2]
```diff
@@ -44,6 +44,11 @@ const REQUIRED_COLOR_PROPS = ACCEPTED_COLOR_PROPS.slice(0, 4);
...
+const { owner, repo } = getRepoInfo(github.context);
```
Comment (rickstaa) [inline scripts/preview-theme.js:50]: Same for OWNER and REPO.

File: scripts/preview-theme.js [hunk 3]
```diff
@@ -561,15 +563,15 @@ export const run = async (prNumber) => {
     debug("Set review state to `REQUEST_CHANGES` and add `invalid` label...");
     if (!DRY_RUN) {
       await addReview(
-        octokit,
+        OCTOKIT,
         pullRequestId,
```
Comment (rickstaa) [inline scripts/preview-theme.js:567]: Here we can then use the uppercase variables.

**Review by rickstaa — CHANGES_REQUESTED** [review]

**Comment by Zo-Bro-23** [comment]
I'm okay with the changes. Can you commit them and push though? I'm outside now, and I don't think I'll be able to do those today.

**Review by rickstaa — APPROVED** [review]

**Comment by rickstaa** [comment]
> I'm okay with the changes. Can you commit them and push though? I'm outside now, and I don't think I'll be able to do those today.

Done feel free to merge.

**Comment by rickstaa** [comment]
Oe conflicts one moment.

**Comment by rickstaa** [comment]
Conflicts are fixed now feel free to merge 👍🏻.

**Pull request merged**